_i18n/en/resources/developer-guides/wallet-rpc.md

```diff
@@ -142,1001 +181,1941 @@ Inputs:
+* *tx_key* - string; transaction secret key.
+* *address* - string; destination public address of the transaction.
 
 Outputs:
 
-* *transfers* - list of:
-  * *amount* - unsigned int; Amount of this transfer.
-  * *spent* - boolean; Indicates if this transfer has been spent.
-  * *global_index* - unsigned int; Mostly internal use, can be ignored by most users.
-  * *tx_hash* - string; Several incoming transfers may share the same hash if they were in the same transaction.
-  * *tx_size* - unsigned int; Size of transaction in bytes.
-  * *subaddr_index* - unsigned int; Subaddress index for incoming transfer.
-  * *key_image* - string; Key image for the incoming transfer's unspent output (empty unless verbose is true).
+* *confirmations* - unsigned int; Number of block mined after the one with the transaction.
+* *in_pool* - boolean; States if the transaction is still in pool or has been added to a block.
+* *received* - unsigned int; Amount of the transaction.
 
-Example (Return "all" transaction types):
+Example:
 
 ```
-$ curl -X POST http://127.0.0.1:18082/json_rpc -d '{"jsonrpc":"2.0","id":"0","method":"incoming_transfers","params":{"transfer_type":"all"}}' -H 'Content-Type: application/json'
-
+$ curl -X POST http://127.0.0.1:18082/json_rpc -d '{"jsonrpc":"2.0","id":"0","method":"check_tx_key","params":{"txid":"19d5089f9469db3d90aca9024dfcb17ce94b948300101c8345a5e9f7257353be","tx_key":"feba662cf8fb6d0d0da18fc9b70ab28e01cc76311278fdd7fe7ab16360762b06","address":"7BnERTpvL5MbCLtj5n9No7J5oE5hHiB3tVCK5cjSvCsYWD2WRJLFuWeKTLiXo5QJqt2ZwUaLy2Vh1Ad51K7FNgqcHgjW85o"}}' -H 'Content-Type: application/json'
 {
   "id": "0",
   "jsonrpc": "2.0",
   "result": {
-    "transfers": [{
-      "amount": 10000000000000,
-      "global_index": 711506,
-      "spent": false,
-      "tx_hash": "&lt;c391089f5b1b02067acc15294e3629a463412af1f1ed0f354113dd4467e4f6c1&gt;",
-      "tx_size": 5870,
-      "subaddr_index": 0,
-      "key_image": ""
-    },{
-      "amount": 300000000000,
-      "global_index": 794232,
-      "spent": false,
-      "tx_hash": "&lt;c391089f5b1b02067acc15294e3629a463412af1f1ed0f354113dd4467e4f6c1&gt;",
-      "tx_size": 5870,
-      "subaddr_index": 0,
-      "key_image": ""
-    },{
-      "amount": 50000000000,
-      "global_index": 213659,
-      "spent": false,
-      "tx_hash": "&lt;c391089f5b1b02067acc15294e3629a463412af1f1ed0f354113dd4467e4f6c1&gt;",
-      "tx_size": 5870,
-      "subaddr_index": 0,
-      "key_image": ""
-    }]
+    "confirmations": 0,
+    "in_pool": false,
+    "received": 1000000000000
   }
 }
 ```
 
-Example (Return "available" transactions):
 
-```
-$ curl -X POST http://127.0.0.1:18082/json_rpc -d '{"jsonrpc":"2.0","id":"0","method":"incoming_transfers","params":{"transfer_type":"available"}}' -H 'Content-Type: application/json'
+### **get_tx_proof**
+
+Get transaction signature to prove it.
+
+Alias: *None*.
+
+Inputs:
+
+* *txid* - string; transaction id.
+* *address* - string; destination publi
```

Comment:
Is this no longer a comma-separated list of account indices?

Reply:
I'm confused. Are you referring to get_tx_proof or get_reserve_proof ? I might have made a mistake anyway.

Reply:
Sorry, those lines referenced above have nothing to do with this comment... I was referring to get_reserve_proof.  I was cross-referencing these notes with my own.  account_index isn't a comma-separated list, it's a number: https://github.com/monero-project/monero/blob/master/src/wallet/wallet_rpc_server.cpp#L2006

Reply:
Ok, good.